### PR TITLE
Work on regional cloud-related yamlfiles for adpsfc prepbufr and lgycld bufr data

### LINKED ISSUE #1048: Work on yaml files for regional prepbufr data
## Description

Use rap prepbufr data as an example to create the yaml file for adpsfc, adpupa, aircar, aircft, and other subsets to convert prepbufr data into ioda v2 NetCDF4 format in support of the regional model (i.e., RRFS) DA activities. Major variables will focus on  cloud, fog, visibility, and gust wind data.

## Requirements
1.  Create yaml files for each subsets and the whole prepbufr file
2.  Use bufr2ioda.x to produce ioda v2 NetCDF4 formatted data files
3.  Use python scripts to make plot to have a preliminary check
4. Make plots by using GSI and ioda data to check if the converted data is correct
5. Test the yaml files within ioda-bundle and validate if the variable names and units pass validation check
6.  Open a pull request to ask the reviewers to review the pull request
7.  The yaml files are merged into the develop branch 

## Acceptance Criteria (Definition of Done)

The yaml filles are merged

## Dependencies

No

### LINKED ISSUE #1049: Work on NASA LaRC cloud products bufr file.
## Description

Create and test yaml file for NASA LaRC cloud bufr data in support the RRFS DA activities

## Requirements
1.  Create yaml files for the bufr file
2.  Use bufr2ioda.x to produce ioda v2 NetCDF4 formatted data file
3.  Use python scripts to make plot to have a preliminary check
4. Make plots by using GSI, original bufr, and ioda data to check if the converted data is correct
5. Test the yaml files within ioda-bundle and validate if the variable names and units pass validation check
6.  Open a pull request to ask the reviewers to review the pull request
7.  The yaml files are merged into the develop branch 

## Acceptance Criteria (Definition of Done)

The yaml file is merged

## Dependencies
No

### PR DESCRIPTION
**Pull request by YoulongXia-NOAA**
## Description

Use rap adpsfc prepbufr and NASA lgycld bufr data as the examples, create the yamlfiles to produce ioda-based netCDF file.

### Issue(s) addressed

Link the issues to be closed with this PR
- fixes #1048
- fixes #1049  

## Acceptance Criteria (Definition of Done)

Passed ctest and the PR will be merged into the develop branch

## Dependencies

This PR dependent on the addition of liquidWaterPath and equivalentBlackBodyTemperature into the ObsSpace.yaml  (ioda#802) -  https://github.com/JCSDA-internal/ioda/issues/802#issuecomment-1271934835

## Impact

No

### CODE REVIEW

File: test/testinput/bufr_ncep_lgycld_rrfs.yaml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+      obsdatain: "./testinput/rap.t06z.lgycld.tm00.bufr_d"
+
+      exports:
+        variables:
+          # MetaData
+          longitude:
+            query: "*/CLONH"
+            transforms:
+              - offset: 360.0
```
Comment (nicholasesposito) [inline test/testinput/bufr_ncep_lgycld_rrfs.yaml:18]: Does this mean that CLONH data is -360 to 0, and it is transformed to 0 +360 ?

Comment (YoulongXia-NOAA) [reply test/testinput/bufr_ncep_lgycld_rrfs.yaml:18]: @nicholasesposito, All values are negative, -80 means 80 oW, if I use 360+(-80) =280, this is correct.  This data for US-alike region.

**Comment by YoulongXia-NOAA** [comment]
> Is this PR dependent on the addition of liquidWaterPath and equivalentBlackBodyTemperature to be in the ObsSpace.yaml so that the tests will pass?

@nicholasesposito, I add the dependency as suggested. However, I think that it does not affect ctest. My ctest has passed. I may need to remove dependency to avoid to block the merging of this PR in near future.

File: test/testinput/bufr_ncep_lgycld_rrfs.yaml (hunk 1 above)
Comment (nicholasesposito) [reply test/testinput/bufr_ncep_lgycld_rrfs.yaml:18]: Ok sounds good.

**Review by nicholasesposito — APPROVED** [review]

**Comment by YoulongXia-NOAA** [comment]
@nicholasesposito, thank you for your time and effort. Well done!

**Review by PatNichols — APPROVED** [review]

**Pull request merged**